awx/ui_next/src/screens/Organization/OrganizationList/OrganizationList.jsx

```diff
@@ -25,64 +26,69 @@ const QS_CONFIG = getQSConfig('organization', {
 function OrganizationsList({ i18n }) {
   const location = useLocation();
   const match = useRouteMatch();
-  const [contentError, setContentError] = useState(null);
-  const [deletionError, setDeletionError] = useState(null);
-  const [hasContentLoading, setHasContentLoading] = useState(true);
-  const [itemCount, setItemCount] = useState(0);
-  const [organizations, setOrganizations] = useState([]);
-  const [orgActions, setOrgActions] = useState(null);
+
   const [selected, setSelected] = useState([]);
+  const [deletionError, setDeletionError] = useState(null);
 
   const addUrl = `${match.url}/add`;
-  const canAdd = orgActions && orgActions.POST;
-  const isAllSelected =
-    selected.length === organizations.length && selected.length > 0;
 
-  const loadOrganizations = async ({ search }) => {
-    const params = parseQueryString(QS_CONFIG, search);
-    setContentError(null);
-    setHasContentLoading(true);
-    try {
-      const [
-        {
-          data: { count, results },
-        },
-        {
-          data: { actions },
-        },
-      ] = await Promise.all([
+  const {
+    result: { organizations = {}, organizationCount, actions },
```

Comment:
I think organizations should default to an empty array.

Reply:
It probably doesn't need a default at all, since I've added the initialValue parameter to the hook